doc/dev/developer_guide/running-tests-locally.rst

```diff
@@ -132,6 +132,31 @@ vstart_runner.py primarily does three things -
     and without admin socket and ``LocalCephCluster`` provides methods to set
     or clear ``ceph.conf``.
 
+Running Workunits Using vstart_enviroment.sh
+--------------------------------------------
+
+Code can be tested by building Ceph locally from source, starting a vstart
+cluster, and running any suite against it.
+Similar to S3-Tests, other workunits can be run against by configuring your enviroment.
+
+Set up the enviroment
+^^^^^^^^^^^^^^^^^^^^^
+
+Configure your enviroment::
+
+    $ . ./build/vstart_enviroment.sh
+    $ cp build/ceph.conf /etc/ceph/
```

Comment:
it'd be nice to include setting up the right ceph.conf as part of vstart_environment.sh, e.g. setting CEPH_CONF to the right path